### PR TITLE
Change coord units

### PR DESCRIPTION
**Pull request by jaymedina**
the sossFieldSim() function [ what generates the simulated field ] only takes the RA and DEC coordinates in unit HH:MM:SS. otherwise, it will return some funky plots. because our form takes in RA and DEC in decimal degrees, I went and converted those units accordingly within the function.

### CODE REVIEW

File: exoctk/contam_visibility/resolve.py [hunk 1]
```diff
@@ -1,8 +1,56 @@
...
+    if round:
+      raS = np.round(((((ra/15)-raH)*60)-raM)*60, 2)
+    else:
+      raS = ((((ra/15)-raH)*60)-raM)*60
+    RA = '{0}{1}:{2}:{3}'.format(rs, raH, raM, raS)
+
+  if ra and dec:
+    return (RA, DEC)
+  else:
+    return RA or DEC
```
Comment (julesfowler) [inline exoctk/contam_visibility/resolve.py]: @jaymedina This is neatly written, but this functionality is already in astropy.coordinates.SkyCoord : https://docs.astropy.org/en/stable/coordinates/.

File: exoctk/contam_visibility/visibilityPA.py [hunk 2]
```diff
@@ -319,10 +318,10 @@ def using_gtvt(ra, dec, instrument, ephFileName=None, output='bokeh'):
...
 
     # Draw the curve and PA min/max patch
-    fig.circle('date', 'panom', color=COLOR, size=1, legend='Nominal Aperture PA',\
-              source=SOURCE, alpha=.5)
-    fig.circle('date', 'pamin', color=COLOR, size=1, source=SOURCE)
-    fig.circle('date', 'pamax', color=COLOR, size=1, source=SOURCE)
+    fig.line('date', 'panom', color=COLOR, legend='Nominal Aperture PA',\
+              source=SOURCE)
+    fig = fill_between(fig, gd, paMin, paMax, color=COLOR, fill_alpha=0.2,\
+                        line_alpha=0.1)
```
Comment (julesfowler) [inline exoctk/contam_visibility/visibilityPA.py]: @jaymedina And like you pointed out, pulling most recent changes should fix this.

File: exoctk/contam_visibility/resolve.py (hunk 1 above)
Comment (jaymedina) [reply exoctk/contam_visibility/resolve.py]: For some reason the code is breaking when I input the RA and DEC coordinates returned using `SkyCoord`. Can we opt to have `SkyCoord` implemented in the next release?

File: exoctk/contam_visibility/resolve.py [hunk 3]
```diff
@@ -1,8 +1,33 @@
...
+    ra_deg = data['RA']
+    dec_deg = data['DEC']
+
+    sc = SkyCoord(ra_deg, dec_deg, unit='deg')
+    ra_hms = '{}:{}:{}'.format(int(sc.ra.hms[0]), \
+                               int(sc.ra.hms[1]), \
+                               np.round(sc.ra.hms[2], 2))
+    dec_hms = '{}:{}:{}'.format(int(sc.dec.dms[0]), \
+                                int(sc.dec.dms[1]), \
+                                np.round(sc.dec.dms[2], 2))
```
Comment (julesfowler) [inline exoctk/contam_visibility/resolve.py]: @jaymedina I thought this was the part you found a one line solution for?

Comment (jaymedina) [reply exoctk/contam_visibility/resolve.py]: @julesfowler nay, I was actually just using that function for testing. the short solution is in `app_exoctk.py`

File: exoctk/exoctk_app/app_exoctk.py [hunk 4]
```diff
@@ -520,13 +521,14 @@ def contam_visibility():
...
             if form.calculate_contam_submit.data:
 
-                # Make field simulation
-
                 # First convert ra and dec to HH:MM:SS
                 ra_deg, dec_deg = float(form.ra.data), float(form.dec.data)
-                ra_hms, dec_hms = deg2HMS(ra_deg, dec_deg)
-
-                contam_cube = fs.sossFieldSim(ra_hms, dec_hms, binComp=form.companion.data)
+                sc = SkyCoord(ra_deg, dec_deg, unit='deg'
```
Comment (jaymedina) [inline exoctk/exoctk_app/app_exoctk.py]: short solution starts here @julesfowler

File: exoctk/contam_visibility/resolve.py (hunk 3 above)
Comment (julesfowler) [reply exoctk/contam_visibility/resolve.py]: Okay. If this was just used for testing, then let's not keep extra code in our final product. Can you zap what isn't being used?

File: exoctk/contam_visibility/visibilityPA.py [hunk 5]
```diff
@@ -16,7 +16,7 @@
 from astropy.table import Table
 from astropy.time import Time
 from bokeh.plotting import figure, ColumnDataSource
-from bokeh.models import HoverTool, ranges
+from bokeh.models import HoverTool
```
Comment (julesfowler) [inline exoctk/contam_visibility/visibilityPA.py]: @jaymedina We need this `ranges` import in order to set limits on the `bokeh` plots.

File: exoctk/contam_visibility/visibilityPA.py [hunk 6]
```diff
@@ -216,8 +216,7 @@ def checkVisPA(ra, dec, targetName=None, ephFileName=None, fig=None):
     fig.xaxis.axis_label = 'Date'
     fig.yaxis.axis_label = 'Position Angle (degrees)'
 
-    #return paGood, paBad, gd, fig
-    return terr_x, terr_y, berr_x, berr_y
+    return paGood, paBad, gd, fig
```
Comment (julesfowler) [inline exoctk/contam_visibility/visibilityPA.py]: @jaymedina Thanks for giving these more useful names.

File: exoctk/contam_visibility/visibilityPA.py [hunk 7]
```diff
@@ -332,7 +331,6 @@ def using_gtvt(ra, dec, instrument, ephFileName=None, output='bokeh'):
     # Plot formatting
     fig.xaxis.axis_label = 'Date'
     fig.yaxis.axis_label = 'Position Angle (degrees)'
-    fig.y_range = ranges.Range1d(0, 360)
```
Comment (julesfowler) [inline exoctk/contam_visibility/visibilityPA.py]: @jaymedina Again -- this was a change we made purposefully -- please bring it back.

**Comment by jaymedina** [comment]
@julesfowler @bourque last test (Mac Python 3.7) failed with line: 
```
No output has been received in the last 10m0s, this potentially indicates a stalled build or something wrong with the build itself.
86Check the details on how to adjust your build configuration on: https://docs.travis-ci.com/user/common-build-problems/#Build-times-out-because-no-output-was-received
87
88The build has been terminated
```

According to the webpage this error links to, it could be a code signing error (?) (`Mac: macOS Sierra (10.12) Code Signing Errors`)

**Comment by julesfowler** [comment]
@jaymedina I will look at your changes now. As far as the travis build,  would recommend making a small insignificant change (like adding a space or line somewhere) to re-trigger the build and see if it sorts itself.

**Pull request merged**